resources/js/Pages/Home.vue

```diff
@@ -49,6 +54,7 @@
                 this.$inertia.get('/results?ids=' + idsToSend)
             },
         },
+        layout: Layout,
```

Comment:
I'm not sure this does anything in our case, since we didn't follow https://inertiajs.com/pages#persistent-layouts (we are still using the layout as a component in our template)

Reply:
You are right, this is not doiing anything. I've removed it.